lib/egree/api_mappers/case_options_mapper.rb

```diff
@@ -35,6 +35,10 @@ def self.map client_key, value, mappers = self.mappers
 
       def self.mappers
         {
+          send_sign_request_email_to_parties: "SendSignRequestEmailToParties",
+          send_finish_email_to_creator: "SendFinishEmailToCreator",
+          send_finish_email_to_parties: "SendFinishEmailToParties",
+          send_recall_email_to_parties: "SendRecallEmailToParties",
           postback_url: "CaseFinishedCallbackUrl",
```

Comment:
We need to change this to `EventCallback` and support the `CaseEventSubscription` structure. It would probably make sense to change the name of our key as well to make it more clear what kinds of postbacks we are talking about.

postback is a bit wierd name as well. Maybe the WebHook nomenclature is better suited?